pages/api/backfill-data.js

```diff
@@ -0,0 +1,50 @@
+import withMethods from '../../util/api/withMethods'
+import {
+  communitiesCount,
+  getAccessList,
+  getAllCommunities,
+  membersCount,
+  updateCommunitiesCounter,
+  updateMembersCounter,
+} from '../../util/fauna'
+
+const ALL = 1000000
+
+const handler = async (req, res) => {
+  if (!process.env.NEXT_PUBLIC_ENABLE_DATA_MIGRATIONS) {
+    res.statusCode = 401
+    return res.json({ success: false, error: 'Not Authorized' })
+  }
+
+  try {
+    const allMembers = await getAccessList(ALL)
+    const allCommunities = await getAllCommunities(ALL)
+
+    const membersCounter = await membersCount()
+    const communitiesCounter = await communitiesCount()
+    const updatedMembersCounter = await updateMembersCounter(
+      membersCounter._id,
+      allMembers.length,
+    )
+    const updatedCommunitiesCounter = await updateCommunitiesCounter(
+      communitiesCounter._id,
+      allCommunities.length,
+    )
+
+    res.statusCode = 201
+    return res.json({
+      success: true,
+      data: {
+        previousMembersCount: membersCounter.counter,
+        previousCommunitiesCount: communitiesCounter.counter,
+        membersCount: updatedMembersCounter.counter,
+        communitiesCount: updatedCommunitiesCounter.counter,
+      },
+    })
+  } catch (error) {
+    res.statusCode = 500
+    return res.json({ success: false, error: error.message })
+  }
+}
+
+export default withMethods(['GET'], handler)
```

Comment:
Do we need to put this behind some form of auth? Or is the ENV var intended to be that control?

Reply:
Another way we might want to tackle this is using single-use migration scripts like this: https://github.com/twos-complement/level-protocol-website/tree/develop/scripts/migrations

Reply:
@royriver23

Reply:
@nicovalencia nice idea, how do you generate the file name? is there something like rails migration creation command or so?

Reply:
@nicovalencia @mckelvey do you guys know of a way to easily use `import` from scripts? I dug into it and it seems `type: "module"` would work, but it's not doing its work. If not, I can convert the endpoint logic into nodejs `require`, but it requires some rewriting of a bunch of stuff (fauna functions, GQL queries, etc), which I don't know would be worth the effort right, given that we could also get into Fauna and run some queries to backfill, thoughts? 

If I should still convert it, I can invest more time but wanted to check with you, first, cause, this is why this task has been delayed.